lib/command/setup_app.rb

```diff
@@ -26,22 +29,69 @@ def call # rubocop:disable Metrics/MethodLength
               "or run 'cpl apply-template #{templates.join(' ')} -a #{config.app}'."
       end
 
+      create_secret_and_policy_if_not_exist unless config.options[:skip_secret_access_binding]
+
       Cpl::Cli.start(["apply-template", *templates, "-a", config.app])
 
-      return if config.options[:skip_secret_access_binding]
+      bind_identity_to_policy unless config.options[:skip_secret_access_binding]
```

Comment:
Just moved the code that was here to a separate `bind_identity_to_policy` method.